EMS/EMS/DataBase.h

```diff
@@ -29,6 +29,6 @@ class DataBase : public IDataBase {
     bool deleteTargets(list<Employee*> targets) override;
 
 private:
-    std::list<Employee*> database_;
+    std::map<int, Employee> database_;
```

Comment:
타입이름을 바꿔서 사번이 key인 것을 강조하면 어떨까요?

```suggestion
using EmployeeID_t = int;
...
class IDataBase {
    ....
    std::map<EmployeeID_t, Employee> database_;
};
```